lib/faktory_worker/job.ex

```diff
@@ -102,4 +102,16 @@ defmodule FaktoryWorker.Job do
     |> to_string()
     |> String.trim_leading("Elixir.")
   end
+
+  defp normalize_job_args(job) do
+    job
```

Comment:
I think you can do this with just `Enum.map`:

```ex
    Enum.map(job, fn
      %_{} = arg ->
        Map.from_struct(arg)

      arg ->
        arg
    end)
```

Reply:
Ah man I don't know why I keep reaching for reduce! Mental note made and now updated